pipenv/core.py

```diff
@@ -351,6 +351,42 @@ def find_a_system_python(line):
     return python_entry
 
 
+def find_python_installer(name, env_var):
```

Comment:
Very good documentation, LGTM but I think `installers.py` is a better place for this function. A little refactor may be required for it.

Reply:
OK, I can make that move, but would prefer to wait for resolution on the other threads of discussion (re: whether this is needed behavior) here before doing it.

Reply:
this seems like a good suggestion that I will agree with and then i would say it's good to go

Reply:
That move has been made, along with some light refactors. Waiting on tests at this moment, if they are green then this is ready for re-review.